### DISCUSSION TITLE
не срабатывает валидация при редактировании поля в таблице

### DISCUSSION THREAD

**Discussion by nikidjan** [totumonline/totum-ru-issues-and-discussions#478] [Неопределенное]
установлена валидация, при добавлении записи - она срабатывает, выдает уведомление.
при редактировании - валидация не работает
что то надо донастроить? или баг?

**Comment by ttm-alexeypolunin** [comment]
У нас не подтверждается

<img width="636" alt="Снимок экрана 2023-11-27 в 17 08 05" src="https://github.com/totumonline/totum-ru-issues-and-discussions/assets/55755565/628f742d-ef0f-479c-9715-9deb30cd8b9f">

**Reply by nikidjan** [reply]
спасибо Алексей, все ок! 
в итоге конечно же тупо регулярка кривая была )

**Reply by ttm-alexeypolunin** [reply]
А, ну это еще хороший вариант, хуже — когда иногда не срабатывает что-то и нихрена непонятно почему.
